frontend/src/components/admin/audit/spec/AuditChangeEntry.spec.tsx

```diff
@@ -1,22 +1,23 @@
-import { fireEvent, render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import '@testing-library/jest-dom'
-import { AuditChange, AuditLog, AuditModelTypes } from '../audit-models';
+import { AuditLog, AuditModelTypes } from '../audit-models';
 import { AuditChangeEntry } from '../AuditChangeEntry';
 
-
-describe('AuditChangeEntry', () => {
-  it('', () => {
-    const change: AuditLog = {
-      date: new Date(),
-      modelId: '5',
-      _id: '',
-      user: '',
-      model: 'client',
-      diff: [],
-    };
-    const modelType: AuditModelTypes = 'client'
-    const rendered = render(<AuditChangeEntry change={change} modelType={modelType} />)
+it.skip('AuditChangeEntry example test', () => {
+  describe('AuditChangeEntry', () => {
```

Comment:
De hierarchy is describe -> it (waarbij describes kunnen genest worden)

Je kan niet echt eerst een `it` doen met daarin een describe...

``` 
describe(
    it(...)
    describe(
      it(...)
    )
) 
```

Je kan puur doen op de bestaande it --> `it.skip`
Or je kan ook doen `xdescribe` en waarschijnlijk ook `describe.skip`

Reply:
Inderdaad, ik had de lege it() binnen de describe over het hoofd gezien. Goed opgemerkt.